### ISSUE TITLE
Is this repo still active?

### ISSUE THREAD

**Issue by Clemapfel** [stetre/moonglfw#7]
I'm considering writing my own glfw wrapper in lua 5.4, and I'm trying to decide if it would be better for me to just start from scratch or to fork this repo. For the latter to be viable, I would need conformation on:
a) If a new issue arises, would they be looked over in a reasonable time-frame (a few days to a week)
b) If I were to open an PR, would there be a dev wiling to do the review?

I know these small repos sometimes are just a one-time thing project and then the devs move on. This isn't a bad thing and I wouldn't fault anybody for it, I'm just trying to see if this is the case here.

Thank you,
C. Cords

**Comment by stetre** [comment]
The repo is still active, and so are all my other repos, with the exception of those explictly archived.
I usually look at issues and PRs fairly soon after their opening, typically in a matter of hours, but I can't give
guarantees that it will always be so (it's a one-man side-project, with all the related limitations and disclaimers).

The bindings are for Lua >= 5.3, thus including 5.4 (5.3 was the current version at the time I wrote it, but I've been
using it with 5.4 since the latter was released).
